doc/man3/OSSL_STORE_INFO.pod

```diff
@@ -122,6 +122,14 @@ given to L<OSSL_STORE_open_file(3)>, the returned name will be the path of
 each object, so if C</foo/bar> was given and that path has the file
 C<cookie.pem>, the name C</foo/bar/cookie.pem> will be returned.
 
+The returned URI is considered canonical and must be unique and permanent
+for the storage where the object (or collection of objects if that's what
+each name represents) resides.
+It is up to each loader to ensure this.
+However, it's possible that certain schemes allow an object (or collection
+thereof) to be reached with alternative URIs; just because one URI is
+canonical doesn't mean that all other variants can't be used.
```

Comment:
But I guess the "doesn't" here and "it's" two lines above don't bother me so much that I want to object to them.